src/warplib/implementations/external_input_checks/external_input_checks_address.ts

```diff
@@ -0,0 +1,22 @@
+import { generateFile, mask } from '../../utils';
+
+const INDENT = ' '.repeat(4);
+
+const import_strings: string[] = [
+  'from starkware.cairo.common.math_cmp import is_le_felt',
+  'from starkware.cairo.common.uint256 import Uint256',
+];
+
+const AddressBoundChecker: Array<string> = [
```

Comment:
How come this is in a warplib file gen file rather that being handwritten cairo when there's only one function generated?